terra-jupyter-aou/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.0.9
```

Comment:
this is a minor version bump...is it intended?

Reply:
yes the minor bump is for gatk only. we bumped aou as a patch https://github.com/DataBiosphere/terra-docker/pull/316/files#diff-111700937e15194bfcc732196ff2c66042b8496f9077dc25383577a79e174092R1

we were always going to bump gatk as a minor bump because nextflow is a big/involved tool that we are now offering in gatk which we did not previously offer in that image